### ISSUE TITLE
Code causes the IDE to crash

### ISSUE THREAD

**Issue by disarray2077** [beefytech/Beef#1606]
The below code (a minimal repro) causes the IDE to crash (apparently only after cleaning the build?).
```cs
using System;

namespace test8
{
	class Tweener<TTarget>
	{
		typealias TweenInit<TProperty> = (int startTime, int endTime, PropertyType<TTarget, TProperty>.Result from, PropertyType<TTarget, TProperty>.Result to, EasingTypes easing);
		
		public void AddTweens<TProperty>(TProperty property, params Span<TweenInit<TProperty>> tweens)
			where TProperty : const String
		{
		}
	}
}
```
Tested with: https://github.com/beefytech/Beef/commit/8c24eb9278a8c0365d83606e3da208f0569064ba

**Comment by disarray2077** [comment]
I updated the repro code because I don't think it was crashing with the code I had posted before, sorry.

**Comment by bfiete** [comment]
Fixed at 1db184e7d87f16a86c9f16add7d17bcfe3e169a5
